snoop/data/collections.py

```diff
@@ -323,20 +341,21 @@ def drop_db(db_name):
 
 
 def create_databases():
-    """Go through [snoop.data.collections.ALL][] and create databases that don't exist yet."""
+    """Go through [snoop.data.collections.list_keys()][] and create databases that don't exist yet."""
 
     dbs = all_collection_dbs()
-    for col in ALL.values():
+    for col_name in list_keys():
```

Comment:
simplify into ` for col in get_all():` and use `col.name` instead of `col_name`?